examples/demo_all.pp

```diff
@@ -31,10 +31,12 @@
   include ciscopuppet::demo_command_config
   include ciscopuppet::demo_domain
   include ciscopuppet::demo_interface
+  include ciscopuppet::demo_interface_portchannel
   #include ciscopuppet::demo_interface_service_vni
   include ciscopuppet::demo_ntp
   include ciscopuppet::demo_ospf
   include ciscopuppet::demo_patching
+  include ciscopuppet::demo_portchannel_global
```

Comment:
Please also include results from running the demo_all.pp for all portchannel functionality.  Run it 2 times to make sure there are not idempotency issues.